cross_list_courses/templates/add_new.html

```diff
@@ -107,10 +107,43 @@ <h3>
             "bInfo" : false,
             "aoColumnDefs": [ { "orderable": false, "targets": 2 } ],
             "oLanguage": {
-              "sSearch": "Search"
+              "sSearch": "Filter"
             }
         });
     });
+
+    $(document).ready(function(){
+        var location_input=$('input[id="autocomplete-pcourse"]');
+        location_input.autocomplete({
+            source: "get_ci_data",
+          minLength: 2
+        });
+      } );
+
+    //   keeps same width as box
+      jQuery.ui.autocomplete.prototype._resizeMenu = function () {
+          var ul = this.menu.element;
+          ul.outerWidth(this.element.outerWidth());
+        }
+
+    $(document).ready(function(){
+            var location_input=$('input[id="primary-course"]');
+            location_input.autocomplete({
+                source: "get_ci_data",
+              minLength: 2
+            });
+          } );
+
+
+    $(document).ready(function(){
+            var location_input=$('input[id="secondary-course"]');
```

Comment:
Should these vars be named uniquely? Not sure how JS handles this type of stuff during the .ready() call